lib/leaf/match.js

```diff
@@ -0,0 +1,32 @@
+module.exports = function( property, value, extra ) {
+  if( !property || !value) {
+    return null;
+  }
+
+  const query = {
+    match: {
+      [property]: {
+        query: value
+      }
+    }
+  };
+
+  const extra_params = ['boost',
```

Comment:
can we define this array out of the function scope to avoid re-allocating memory on every invocation?

also, unusual formatting 🤷‍♂ almost missed the first element

Reply:
I can't believe that V8 wouldn't be able to perform this optimization itself, and either way, I feel like the readability benefit of having the array close to where it is used outweighs any potential performance changes, so I'd like to leave this code as is.